pkgs/by-name/yt/ytui-music/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, sqlite
+, stdenv
+, darwin
+, mpv
+, youtube-dl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ytui-music-unstable";
+  # Upstream doesn't currently have tags that would be any more meaningful than date of commit
+  # Change this when they upstream has a proper release
+  version = "18-09-2023";
+
+  src = fetchFromGitHub {
+    owner = "sudipghimire533";
+    repo = "ytui-music";
+    rev = version;
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/256702/commits/66733f9113ecfd9f63910008c2056bd3767d3676